task-5/index.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { useEffect, useState } from 'react';
+
+function App() {
+
+    var [fuel, setFeul] = useState(0);

```

Comment:
```suggestion
    const [fuel, setFuel] = useState<number | null>(null);
```
Typo and let's default to null as we don't have the actual number yet and 0 is a perfectly valid value the user would have. Also, use const/let instead of var.